src/FlightDisplay/FlightDisplayViewUVC.qml

```diff
@@ -47,12 +48,16 @@ Rectangle {
         source:         camera
         anchors.fill:   parent
         fillMode:       VideoOutput.PreserveAspectCrop
-        visible:        !QGroundControl.videoManager.isGStreamer
-    }
-    onVisibleChanged: {
-        if(visible)
-            camera.start()
-        else
-            camera.stop()
+        visible:        QGroundControl.videoManager.isUvc
+
+        onVisibleChanged: {
+            console.log('UVC Video output visible: ', visible);
+            if (visible) {
+                camera.start()
+                }
```

Comment:
The indentation is wrong

Reply:
@patrickelectric identation is good because I've pushed `onVisibleChanged` into `VideoOutput` element since this event is a part of `VideoOutput`.
Please unfold lines 40-50 to see it.